bigbluebutton-html5/private/locales/en.json

```diff
@@ -25,8 +25,8 @@
   "app.userlist.menu.kickUser.label": "Kick user",
   "app.userlist.menu.muteUserAudio.label": "Mute user",
   "app.userlist.menu.unmuteUserAudio.label": "Unmute user",
-  "app.userlist.useritem.nostatus.arialabel": "{username} {presenter} {you}",
-  "app.userlist.useritem.status.arialabel": "{username} {presenter} {you} current status {status}",
+  "app.userlist.useritem.nostatus.arialabel": "{0} {1} {2}",
+  "app.userlist.useritem.status.arialabel": "{0} {1} {2} current status {3}",
```

Comment:
We have the same in the Flash client, just follow the same structure. These are the names of the dynamic variables, which are injected on the fly.

There is nothing to translate here - translators will translate words only and keep the same numbers and brackets :-)